pom.xml

```diff
@@ -12,4 +18,25 @@
     <version>0.0.1-SNAPSHOT</version>
     <name>champmaker-auth</name>
     <description>Authentication service for Champmaker application</description>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-validation</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```

Comment:
I think we can work without Lombok in this project. Let's remove this dependency.